lib/src/connection.class.ts

```diff
@@ -0,0 +1,28 @@
+import { BaseHeader } from "./base-header.class";
+
+export class Connection extends BaseHeader {
+  protected _connection: string;
+
+  constructor(connection: string) {
+    super();
+    this._connection = connection;
+  }
+
+  public getConnection(): string {
+    return this._connection;
+  }
+  public setConnection(connection: string) {
```

Comment:
```suggestion
  public setConnection(connection: 'keep-alive' | 'close') {
```

Nice, can we add some more specific types here if there are only set values that are allowed?